data-processor/src/main/java/io/micronaut/data/processor/visitors/finders/DtoJoinPathsParser.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.data.processor.visitors.finders;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.micronaut.inject.ast.ClassElement;
+import io.micronaut.inject.ast.PropertyElement;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author Sergio del Amo
+ */
+public class DtoJoinPathsParser {
```

Comment:
Not sure of the point of this class, since there is only one method it can probably just exist in the parent abstract class

Reply:
I've moved this to the abstract class.